int32 trajectory_id
int32 submap_index
---
int32[] connections
